spec/helpers/decorate_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Tramway::Helpers::DecorateHelper, type: :controller do
+  let(:controller) { ActionController::Base.new }
+
+  before do
+    controller.extend described_class
+  end
+
+  describe '#tramway_decorate' do
+    context 'with success' do
+      context 'with default decorator' do
+        it 'decorates object' do
```

Comment:
because there is no need to create one more nested context here, and context `with default decorator` has 2 tests. One of them needs `user` object and the second one needs `users`. So, there are 3 ways:
1. Create 2 more nested contexts each of them will contain just one test.
2. Add two `let`s and user and users will be created 2 times
3. Create needed variable inside the test

I've chosen third one :slightly_smiling_face: